ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-stable-4.13.yaml

```diff
@@ -0,0 +1,108 @@
+base_images:
+  ansible:
+    name: "4.13"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.13"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.13"
+  tools:
+    name: "4.13"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      stream: 4-dev-preview
+      version_bounds:
+        lower: 4.13.0-0
+        upper: 4.14.0-0
+  multi-latest:
```

Comment:
you'd test by removing this stanza and using multi in the architecture field above.

Reply:
Per what you suggested in last meeting, you'd prefer to minimum the files. So I was creating the file to hold all the upgrade to multi-arch 4.13 from both single-arch and multi-arch stable 4.13. If you'd prefer to add a separate file to hold single-arch 4.13 to multi-arch 4.13, it's fine with me updating based on your comments.

Reply:
Hi @shellyyang1989, my suggestion is to define single-arch to multi-arch upgrade/migration testing in the related single-arch test config. For example, arm64-to-multi can be defined in the arm64 upgrade test config files.

Reply:
It's fine with me!